### PR TITLE
Add Styleguidist docs for MediaPlayer component

### PR DESCRIPTION
**Pull request by Dananji**
`MediaPlayer` component is not included in the Styleguidist docs. 
This PR includes changes in the code with some re-structuring in  `src/components/MediaPlayer` directory to facilitate this.

Screenshot from 2023-06-20 17-02-23

Merge after #199

### CODE REVIEW

File: public/prod/lunchroom_manners.json [hunk 1]
```diff
@@ -257,7 +257,7 @@
                   ]
                 }
               ],
-              "target": "/prod/lunchroom_manners/canvas/1"
+              "target": "https://iiif-react-media-player.netlify.app//prod/lunchroom_manners/canvas/1"
```
Comment (masaball) [inline public/prod/lunchroom_manners.json:260]: Looks like there's an extra slash here and at ln 284

```suggestion
              "target": "https://iiif-react-media-player.netlify.app/prod/lunchroom_manners/canvas/1"
```

File: public/prod/lunchroom_manners.json [hunk 2]
```diff
@@ -281,7 +281,7 @@
                   ]
                 }
               },
-              "target": "/prod/lunchroom_manners/canvas/1"
+              "target": "https://iiif-react-media-player.netlify.app//prod/lunchroom_manners/canvas/1"
```
Comment (masaball) [inline public/prod/lunchroom_manners.json:284]:
```suggestion
              "target": "https://iiif-react-media-player.netlify.app/prod/lunchroom_manners/canvas/1"
```

**Review by masaball — COMMENTED** [review]
Saw a minor typo in a couple of the URIs, but once those are fixed I think it's good to merge.

File: public/prod/lunchroom_manners.json (hunk 1 above)
Comment (Dananji) [reply public/prod/lunchroom_manners.json:260]: Good catch :100:

**Pull request merged**